packages/@adobe/spectrum-css-temp/components/modal/index.css

```diff
@@ -38,11 +38,9 @@ governing permissions and limitations under the License.
   width: 100vw;
   /* On mobile browsers, vh units are fixed based on the maximum height of the screen.
    * However, when you scroll, the toolbar and address bar shrink, making the viewport resize.
-   * We use the fill-available value to counteract this where supported. */
+   * We use the stretch value to counteract this where supported. And rely on autoprefixer for prefixes. */
   height: 100vh;
-  height: -moz-available;
-  height: -webkit-fill-available;
-  height: fill-available;
+  height: stretch;
```

Comment:
Do the prefixes get applied by postcss?

Reply:
it looks like webkit is, but moz is not. It seems to be a bug in that it isn't picked up by specifying firefox versions, and instead only gets added when the global previous versions syntax is used. According to caniuse, i think i should add that one.
![Screen Shot 2020-06-26 at 12 47 52 PM](https://user-images.githubusercontent.com/698229/85895356-66c99900-b7ab-11ea-8322-67ea3a3d2f44.png)